apps/ledger-live-desktop/src/renderer/modals/Receive/steps/StepReceiveFunds.tsx

```diff
@@ -103,6 +104,11 @@ const Receive1ShareAddress = ({
           <UTXOAddressAlert />
         </Box>
       )}
+      <FeatureToggle featureId="lldMemoTag">
```

Comment:
you can use the feature flag selector for this one

Reply:
answered above

Reply:
the wrapper already use it